CHANGELOG.md

```diff
@@ -0,0 +1,491 @@
+
+## Multi-view forever!
+
+Now you can open ALL THE PROFILES! and move between them with your mouse and keyboard shortcuts without it turning multiview off. Have fun multiplaying!
+
+![](https://www.mudlet.org/wp-content/uploads/2020/11/Selection_368-1024x446.png)
+
+## Telnet CHARSET support
+
+Game admins will be happy about this feature – it is now possible to automatically set Mudlet’s text encoding – for example, to be utf8 in case your game supports emojis – using the [CHARSET protocol](https://tools.ietf.org/html/rfc2066). Big thanks to Tamarindo from [StickMUD](https://www.stickmud.com/) for pulling that one in.
+
+## Music/sound in MXP
+
+Tamarindo also went ahead and added sound/music support to Mudlet’s MXP implementation. That said, there’s a much more modern [Mud Client Media Protocol](https://wiki.mudlet.org/w/Special:MyLanguage/Standards:MUD_Client_Media_Protocol) that you should aim to use 😉
+
+## Scrollbar improvements
+
+You’ll not be surprised to hear this one is Edru as well. Had you noticed Mudlet’s scrollbars behaving a bit… oddly? Jumping around, not resizing itself, stuff like that? Well, Edru’s put a lot of work into this and it should all work much more smoothly now.
+ 
+
+## macOS 10.13 High Sierra
+
+Apple will stop supporting 10.13 High Sierra in November 2020, and Mudlet will have to follow suit. Thus Mudlet 4.10 will be the last version to support 10.13, which had a good run of 3 years!
+
+## Credits
+
+Thanks to all coders that have contributed to this massive release: Andre Castellanos, Damian Monogue, Eraene, Fae, Gustavo Sousa, Ian Adkins, Jonathan Mohrbacher, Kebap, keneanung, Manuel Wegmann, Matthias Urlichs, Mike Conley, Stack, Stephen Lyons, and Vadim Peretokin.
+
+Thanks to all translators to translated Mudlet into their language: Alan Sneath (alsneath), Anubisa, emeraldboy, Hsin Hsiang Peng (Hsins), Kae (itsTheFae), Leris, Marco “M0lid3us” Tironi (wiploo), rodeos, Vadim Peretokin (vperetokin), vingi, and 王AQ (Anselmus).
+
+This massive piece of work would not be possible without all of you :)
+
+## Changelog
+
+**added:**
+
+- ability to display room names on the map (no auto-align them yet)
+- an event to catch all events – `*`
+- background image for miniconsoles is now possible, same as labels
+- background image for the main window is now possible
+- English spellchecking dictionaries are now included
+- game admins can now set encoding via telnet CHARSET (so utf8 for emojis)
+- miniconsoles can now have commandlines in them
+- music is now supported in MXP
+- option not to highlight text (history) when pressing the ↑ button
+- option to use your own pathfinding algorithm instead of the built-in one
+- you can now create standalone commandlines now allow you to ask the player for info
+- transparency in now supported by cecho/decho/hecho
```

Comment:
If we are sorting these alphabetically then this needs to move up one line!

Reply:
I don't think we are going to though